acquia_lift.module

```diff
@@ -618,23 +587,6 @@ function acquia_lift_personalize_agent_status_presave($agent_name, $status) {
 }
 
 /**
- * Implements hook_personalize_agent_update_status().
- */
-function acquia_lift_personalize_agent_update_status($agent_name, $old_status, $status) {
-  // Clear the agent's verification status.
-  acquia_lift_agent_clear_verified_status($agent_name);
```

Comment:
Oh right, what I was thinking was that we'd just entirely remove the ability to change agent status from the unibar, but I forgot to actually do that part :/